biom/assets/exercise_api.py

```diff
@@ -0,0 +1,105 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2011-2015, The BIOM Format Development Team.
```

Comment:
@wasade what is the goal of this file? I understand that it is exercising the API, but given that those calls are part of the Table object, shouldn't those functions be tested using the unit-tests of the Table object? Or am I missing something?